packages/service-clients/end-to-end-tests/azure-client/src/test/containerCreate.spec.ts

```diff
@@ -256,4 +259,118 @@ for (const testOpts of testMatrix) {
 			assert(featureGates.length > 0);
 		});
 	});
+
+	/**
+	 * Testing scenarios for creating/loading containers with the legacy (LTS) version of AzureClient.
+	 */
+	describe(`Container create with legacy version (${testOpts.variant})`, () => {
+		const connectTimeoutMs = 10_000;
+		const isEphemeral: boolean = testOpts.options.isEphemeral;
+		let clientCurrent: AzureClient;
+		let clientLegacy: AzureClientLegacy;
+		const schemaCurrent = {
+			initialObjects: {
+				map1: SharedMap,
+			},
+		} satisfies ContainerSchema;
+
+		const schemaLegacy = {
+			initialObjects: {
+				map1: SharedMapLegacy,
+			},
+		};
+
+		beforeEach("createAzureClients", function () {
+			clientCurrent = createAzureClient();
+			clientLegacy = createAzureClientLegacy();
+			if (isEphemeral) {
+				// TODO: Should we skip ephemeral tests for legacy clients?
```

Comment:
I think skipping is probably fine here for now.  Longer-term, when we add ephemeral support to AzureClient we'll probably want to cover it, but since it's not a public API currently I think we can skip.